tests/dir_helpers.py

```diff
@@ -212,45 +226,19 @@ def _git_init():
     #
     #    GitCommandNotFound: Cmd('git') not found due to:
     #        OSError('[Errno 35] Resource temporarily unavailable')
-    git = retry(5, GitCommandNotFound)(Repo.init)()
+    git = retry(5, GitCommandNotFound)(Repo.init)(path)
     git.close()
 
 
 @pytest.fixture
-def dvc(tmp_dir, request):
-    from dvc.repo import Repo
-
-    if "scm" in request.fixturenames:
-        if not hasattr(tmp_dir, "scm"):
-            _git_init()
-
-        dvc = Repo.init(fspath(tmp_dir))
-        dvc.scm.commit("init dvc")
-    else:
-        dvc = Repo.init(fspath(tmp_dir), no_scm=True)
-
-    try:
-        tmp_dir.dvc = dvc
-        yield dvc
-    finally:
-        dvc.close()
-
-
-@pytest.fixture
-def repo_template(tmp_dir):
-    tmp_dir.gen(REPO_TEMPLATE)
-
-
-@pytest.fixture
-def run_copy(tmp_dir, dvc, request):
+def run_copy(tmp_dir, dvc):
     tmp_dir.gen(
         "copy.py",
         "import sys, shutil\nshutil.copyfile(sys.argv[1], sys.argv[2])",
     )
 
     # Do we need this?
```

Comment:
It is a leftover from your previous PRs that slipped-by, but is this still relevant or were you able to figure out? 🙂

Reply:
This is a part of this PR

Reply:
@Suor `    # Do we need this?` is a leftover that has been there before. Do we need it anymore?

Reply:
Ah, not sure. But this question is independent from this PR )

Reply:
Removing it doesn't break any tests though, it might if we will `run_copy` in context of some branches.

Reply:
Removed it. Let's hope we won't need it)